contracts/bridge-v1.yml

```diff
@@ -1230,4 +1626,176 @@ components:
+                    type: object
+                    required:
+                      - patientReference
+                      - careContextReference
+                    properties:
+                      patientReference:
+                        type: string
+                        example: batman@tmh
+                      careContextReference:
+                        type: string
+                        example: Episode1
+                purpose:
+                  $ref: '#/components/schemas/UsePurpose'
+                hip:
+                  type: object
+                  allOf:
+                    - $ref: '#/components/schemas/OrganizationRepresentation'
+                    - xml:
+                        name: hip
+                hiu:
+                  type: object
+                  allOf:
+                    - $ref: '#/components/schemas/OrganizationRepresentation'
+                    - xml:
+                        name: hiu
+                consentManager:
+                  type: object
+                  allOf:
+                    - $ref: '#/components/schemas/OrganizationRepresentation'
+                    - xml:
+                        name: consentManager
+                requester:
+                  $ref: '#/components/schemas/Requester'
+                hiTypes:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/HITypeEnum'
+                permission:
+                  $ref: '#/components/schemas/Permission'
+            signature:
+              type: string
+              example: Signature of CM as defined in W3C standards; Base64 encoded
+        error: 
+          $ref: '#/components/schemas/Error'
+        resp:
+          $ref: '#/components/schemas/RequestReference'
+    Requester:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: Dr. Manju
+        identifier:
+          type: object
+          properties:
+            type: 
+              type: string
+              example: REGNO
+            value:
+              type: string
+              example: "MH1001"
+            system:
+              type: string
+              example: https://www.mciindia.org
+    HIPHealthInformationRequest:
+      type: object
+      required:
+        - requestId
+        - timestamp
+        - transactionId
+        - consent
+        - dataPushUrl
+        - keyMaterial
+        - dateRange
+      properties:
+        requestId:
+          $ref: '#/components/schemas/UuidSchema'
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        transactionId:
+          $ref: '#/components/schemas/UuidSchema'
+        consent:
+          $ref: '#/components/schemas/consent'
+        dateRange:
+          $ref: '#/components/schemas/dateRange'
+        dataPushUrl:
+          type: string
+        keyMaterial:
+          $ref: '#/components/schemas/keyMaterial'
```

Comment:
I think once saurubh asked us remove the Curve. Not sure about expiry inside public key and algorithm. Check with him please.